### ISSUE TITLE
Plugin can block IDE at project load with MFA modal

### ISSUE THREAD

**Issue by hardbyte** [aws/aws-toolkit-jetbrains#2428]
The plugin shouldn't block the entire IDE from starting just because the current AWS profile requires an MFA token.

The IDE brings up an MFA entry box and doesn't load project files until it has been dealt with.

I suspect it may be due to using a `role_arn` in my AWS config. I have something like the following:

```
[profile company-dev]
region = ap-southeast-2
role_arn = arn:aws:iam::XXXX:role/admin-role
source_profile = company-central
mfa_serial = arn:aws:iam::XXXXX:mfa/bthorne
role_session_name = bthorne
duration_seconds = 14400
```

Then in my .aws/credentials there is an entry for `company-central`.

**Expected behavior**
I don't think any plugin should ever block the entire IDE before a project is loaded.

**Screenshots**

![image](https://user-images.githubusercontent.com/855189/107422080-89b08e00-6b7f-11eb-84a3-25d5cc86b90a.png)

After entering a correct MFA I still have to select credentials and enter my MFA token again:

![image](https://user-images.githubusercontent.com/855189/107422252-a8168980-6b7f-11eb-8135-e3fbd22873a4.png)

**Your Environment**
- OS: Archlinux
- SAM CLI version:
```
aws --version
aws-cli/1.18.223 Python/3.9.1 Linux/5.10.13-arch1-2 botocore/1.19.63
```
- JVM/Python version: Python 3.9 in the current project
- PyCharm:
```
PyCharm 2020.3.3 (Professional Edition)
Build #PY-203.7148.72, built on January 27, 2021
Licensed to Brian Thorne
Subscription is active until May 3, 2021.
Runtime version: 11.0.9.1+11-b1145.77 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.10.13-arch1-2
GC: ParNew, ConcurrentMarkSweep
Memory: 4029M
Cores: 8
Registry: ide.tooltip.initialDelay=668
Non-Bundled Plugins: com.intellij.ideolog, org.intellij.plugins.hcl, org.jetbrains.plugins.go-template, zielu.gittoolbox, mobi.hsz.idea.gitignore, org.toml.lang, io.plugin.tsnode, com.koxudaxi.poetry, com.koxudaxi.pydantic, aws.toolkit, ru.adelf.idea.dotenv, org.rust.lang
Current Desktop: GNOME
```

**Comment by abrooksv** [comment]
I am sorry this is happening, we are supposed to disable credential validation on startup when the credentials have MFA.

I wonder if it is the same bug fixed by #2374. This build has the fix in it, if you want to try it: https://plugins.jetbrains.com/plugin/11349-aws-toolkit/versions/eap/110247

In the plugins preferences page, select Install from disk in the Gear menu, and select the zip file

**Comment by abrooksv** [comment]
We released 1.24 today that removes an AWS call that could be triggered on start up, can you let us know if you are still seeing this issue? We were not able to reproduce it.

**Comment by hardbyte** [comment]
Thanks @abrooksv - I haven't run into that modal since upgrading to 1.24.
